converter/babeltrace.c

```diff
@@ -175,6 +189,10 @@ static void usage(FILE *fp)
 	fprintf(fp, "      --clock-gmt                Print clock in GMT time zone (default: local time zone)\n");
 	fprintf(fp, "      --clock-force-correlate    Assume that clocks are inherently correlated\n");
 	fprintf(fp, "                                 across traces.\n");
+#ifdef ENABLE_DEBUGINFO
+	fprintf(fp, "      --debug-dir                Directory in which to look for debugging information\n");
+	fprintf(fp, "                                 files during debuginfo analysis. (default: /usr/lib/debug/)\n");
```

Comment:
So I guess it's just a question of whether we want to be consistent with how GDB behaves, or provide a default that accommodates users with different `$prefix` values.